About.html

```diff
@@ -0,0 +1,136 @@
+                    <li><a class="Desktopmenu-item" href="">BuyNow</a></li>
+                    <li><a class="Desktopmenu-item" href="">Contact</a></li>
+                    <li>
+                        <span id="unique-item">BookStore</span>
+                    </li>
+                </ul>
+            </div>
+            <!-- End Desktop -->
+
+        </nav>
+    </header>
+
+    <section id="home" class="Aboutus">
+        <img class="logo" src="./img/logo.png" alt="Logo">
+            <h1 class="notshowondesktop">
+                <span>"Hello! Pakistani People"</span><br>
+                Pakistan's Largest Online Book Store! <span class="numberclass">2020</span>
+            </h1>
+            <h1 class="Desktop-text">
+                Pakistan's Largest <br>Online Book Store!
+            </h1>
+            <div class="About-text">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Id cumque neque qui rerum consequuntur. Illo
+                culpa
+                officia aliquid illum cumque?
+            </div>
+            <div class="About-text-Desktop">
+                Lorem ipsum, dolor sit amet consectetur adipisicing elit. Doloremque corporis quam ducimus, natus
+                explicabo debitis alias enim veritatis, laudantium, beatae voluptate ipsum numquam? Optio earum, odit
+                accusantium ducimus debitis ipsam.lorem20 Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Soluta dignissimos atque vitae consequuntur laborum voluptatem, non
+            </div>
+            <div class="About-text2">
+                Please Contact us on our Email for any further questions about The Book Store!<br><span
+                    class="about-Email">waleedamjad99@hotmail.com</span>
+            </div>
+    </section>
+
+    <section class="aboutSection2">
+
+        <div class="About-text3">
+            <h2>The Book Store <br> Pakistan Store Logo!</h2>
+            </div>
+        <div class="aboutline"></div>
+        <div class="About-text4">
+            The logo of Book Store was decided in back january, 2022 through the logo competition held in paris on
+            17.jan.2022
+        </div>
+
+        <div class="About-logo">
+            <img src="./img/footerlogo.png" alt="LOGO" width="200">
+        </div>
+    </section>
+
+    <section class="aboutSection2">
+
+        <div class="About-text3">
+            <h4>See the past of <br> The Book Store!</h4>
+        </div> 
+        <div class="aboutline"></div>
+        <div class="pastimagestext">
+            Take a look at the last two Biggest Book festivals whicht Took place in London and in Paris.
+        </div>
+        <div class="pastimages">
+            <img src="./img/2018.png" alt="Pastfunctionimg1" width="350">
+            <img src="./img/2020.png" alt="Pastfunctionimg1" width="350">
+        </div>
+    </section>
```

Comment:
- [ ] On the about page, kindly add some margin to the past image as the images are overlapping, and add some between the text and the images.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/169385739-7c5b794e-abb1-44f0-95d0-09afd17ce512.PNG)

Behance template
![figma](https://user-images.githubusercontent.com/85095702/169385929-0404ce48-0c82-478d-84e3-1f3e7a716f9e.PNG)